### DISCUSSION TITLE
How to hide specific property in response

### DISCUSSION THREAD

**Discussion by xinhash** [TypedProject/ts-express-decorators#1482] [Q&A] [answered]
Hi I've a User model as defined below and I want to hide password in response for all cases. How to do so ?

```typescript
enum Roles {
  SUPERADMIN = "superadmin",
  ADMIN = "admin",
  TEACHER = "teacher",
  STUDENT = "student",
}

export class User {
  @Groups("!creation")
  @ObjectID("id")
  _id: string;

  @MinLength(3)
  @MaxLength(50)
  @Trim()
  username: string;

  @Indexed()
  @Required()
  @Email()
  @Unique()
  @Trim()
  email: string;

  @Groups("creation")
  @Required()
  @MinLength(4)
  @MaxLength(20)
  password: string;

  @Enum(Roles)
  @Default("admin")
  role: Roles;

  @Optional()
  @Default(true)
  isActive: boolean;

  @Optional()
  @Default(true)
  isVerified: boolean;

  token: string;
}

```

Repo: https://github.com/xinhash/tsed-mongo-jwt-auth.git

**Comment by Romakita** [comment, marked as answer]
Using group the password won’t be serialised.

**Reply by Romakita** [reply]
This scenario is described in the documentation

**Reply by Romakita** [reply]
https://tsed.io/docs/model.html#groups

**Reply by xinhash** [reply]
@Romakita I'm still facing issues. Can you please check repo for the same https://github.com/xinhash/tsed-mongo-jwt-auth.git
Works as expected: http://0.0.0.0:8083/v3/docs/#/AuthController/authControllerLogin
Doesn't work: http://0.0.0.0:8083/v3/docs/#/AuthController/authControllerGetUserInfo. Password still present in response.
```json
{
  "id": "610bf044e1cd7204050aaeaa",
  "username": "superadmin",
  "email": "superadmin@example.com",
  "password": "$argon2i$v=19$m=4096,t=3,p=1$fpzePRb3PNPNtdzi5c+ki9ljMF/rIjOoCY9M5/chk3M$L4GD+JrXukxEqkJbsxiIP8PX/nwdBIv6Il3b3w8hgpc",
  "role": "superadmin",
  "isActive": true,
  "isVerified": true
}
```

**Reply by Romakita** [reply]
Hello @xinhash 
A release to fix your issue is on going. Sorry for the late answers ;)
